src/Page.js

```diff
@@ -822,10 +834,13 @@ class Page {
           this.collectAllPageSections();
           this.buildPageNav();
 
-          return fs.outputFileAsync(this.resultPath, htmlBeautify(
-            this.template.render(this.prepareTemplateData()),
-            Page.htmlBeautifyOptions,
-          ));
+          const renderedTemplate = this.template.render(this.prepareTemplateData());
+          const outputTemplateHTML
+            = this.jsBeautify
+              ? htmlBeautify(renderedTemplate, Page.htmlBeautifyOptions)
```

Comment:
https://github.com/beautify-web/js-beautify/blob/master/js/src/html/options.js
The `htmlBeautifyOptions` will need to be updated with `textarea` as well.